CMakeLists.txt

```
@@ -181,6 +183,9 @@ find_package2(PUBLIC ROOT REQUIRED
   COMPONENTS ${required_root_components}
   OPTIONAL_COMPONENTS ${optional_root_components})
 include(ROOTMacros)
+
+fairroot_check_root_cxxstd()
```

```suggestion
fairroot_check_root_cxxstd_compatibility()
```

Somehow, I was missing what the check is about. What do you think?

---

Yeah, maybe that name is better! Next push will have the new name.

Do we need more docs on this?

---

Some inline comments above each function with a brief description would be good style, but we can add it, when we migrate it to FairCMakeModules